src/identifierDfa.cpp

```diff
@@ -0,0 +1,33 @@
+#include "identifierDfa.h"
+#include "states.h"
+
+int identifierError(char c) {
+    return DS_ERROR;
+}
+
+int isJavaLetter(char c) {
+    if(('a' <= c && c <= 'z') || ('A' <= c && c <= 'Z') || c == '_' || c == '$') {
+        return DS_ACCEPT;
+    }
+
+    return DS_ERROR;
+}
+
+int isJavaLetterOrDigit(char c) {
+    if(('a' <= c && c <= 'z') || ('A' <= c && c <= 'Z') || ('0' <= c && c <= '9') 
+            || c == '_' || c == '$') {
```

Comment:
('a' <= c && c <= 'z') || ('A' <= c && c <= 'Z') || ('0' <= c && c <= '9') ||
c == '_' || c == '$')

indentation and stylinggs